src/test.rs

```diff
@@ -43,9 +43,9 @@ impl JackHandler for TestHandler {
 
 #[test]
 fn static_fns() {
-    Client::<TestHandler>::name_size();
-    Port::name_size();
-    Port::type_size();
+    client_name_size();
+    port_name_size();
+    port_type_size();
 }
 
 #[test]
```

Comment:
yikes, didn't realize that I left testing in such a terrible state. I'll work on improving it.